app/models/item.rb

```diff
@@ -1,3 +1,4 @@
 class Item < ActiveRecord::Base
-  belongs_to :bill
+      belongs_to :bill
```

Comment:
[Style]
Please fix the spacing here so that the indentation is 2 spaces, not six.